besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1249,6 +1249,33 @@ public void p2pOptionsRequiresServiceToBeEnabledToml() throws IOException {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
+  @Test
```

Comment:
After looking at the tests in CommandLineUtilsTest, I think we leave these env tests out.